src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Microsoft.CodeAnalysis.LanguageServer.csproj

```diff
@@ -1,6 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
     <PropertyGroup>
         <OutputType>Exe</OutputType>
+        <UseAppHost Condition="'$(RuntimeIdentifier)' == ''">false</UseAppHost>
```

Comment:
This might deserve a comment to explain a bit more. Is this a temporary workaround or really how it should work?

Reply:
Not a temporary workaround.  If we're not building and publishing for a specific runtime identifier, we should be producing platform neutral bits, which they are generally.  However by default since we have an outputtype of exe and we're building on windows, it also produces a .exe which we don't need.

Solution is to tell it not to create an exe if we're not building for a specific runtime identifier.  I'll put a comment to this effect in the code